src/index.ts

```diff
@@ -334,3 +334,39 @@ export async function solve(
     pipPackages
   };
 }
+
+export function showPackagesList(
+  installedPackages: ISolvedPackages,
+  logger: ILogger
+) {
+  if (Object.keys(installedPackages).length) {
+    installedPackages = sort(installedPackages);
+
+    const nameWidth = 30;
+    const versionWidth = 30;
+    const buildWidth = 30;
+
+    logger.log(
+      `${'Name'.padEnd(nameWidth)}${'Version'.padEnd(versionWidth)}${'Build'.padEnd(buildWidth)}`
+    );
+
+    logger.log('─'.repeat(nameWidth + versionWidth + buildWidth));
+
+    Object.keys(installedPackages).forEach(filename => {
+      const text = `${installedPackages[filename].name.padEnd(nameWidth)}${installedPackages[filename].version.padEnd(versionWidth)}${installedPackages[filename].build_string?.padEnd(buildWidth)}`;
+      logger.log(text);
+    });
+  }
+}
+
+export function sort(installed: ISolvedPackages) {
+  const sorted = Object.entries(installed).sort((a, b) => {
+    const packageA: any = a[1];
+    const packageB: any = b[1];
+    return packageA.name.localeCompare(packageB.name);
+  });
+
+  const sortedInstalled = Object.fromEntries(sorted);
```

Comment:
I understand we don't have a guaranty of order of keys with `Objects`, but using a Map would.

From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Objects_and_maps_compared :

> The keys in Map are ordered in a straightforward way: A Map object iterates entries, keys, and values in the order of entry insertion